deprish_to_eia.py

```diff
@@ -164,34 +172,59 @@ def get_fuzzy_matches(deprish_df, mul_df, key_deprish, key_mul, threshold=75):
 
 def match_merge(deprish_df, mul_df, key_deprish, key_mul):
     """Generate fuzzy matches and merge relevant colums from eia."""
-    deprish_cols = [
-        'utility_id_ferc1', 'utility_id_pudl', 'utility_name_ferc1', 'state',
-        'plant_id_pudl', 'plant_name', 'report_year', 'plant_name_match']
-    mul_cols = [
-        'record_id_eia', 'plant_name_new', 'plant_name_eia', 'plant_id_eia',
-        'report_year', 'plant_part', 'generator_id', 'unit_id_pudl',
-        'prime_mover_code', 'energy_source_code_1',
-        'technology_description', 'ferc_acct_name', 'utility_id_eia',
-        'true_gran', 'appro_part_label', 'appro_record_id_eia',
-    ]
     logger.info("Merging fuzzy matches.")
     match_merge = pd.merge(
         get_fuzzy_matches(
             deprish_df, mul_df,
             key_deprish=key_deprish, key_mul=key_mul,
-            threshold=75)[deprish_cols],
+            threshold=75)[DEPRISH_COLS + ['plant_name_match']],
```

Comment:
Up above it looked like the `plant_name_match` column is maybe a mix of strings and `np.nan` (float) values. Do you maybe want to use a homogeneous type instead?